GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift

```diff
@@ -126,7 +126,17 @@ extension SecureConversations {
         // it is single source of data (including styles)
         // for the data-driven view.
         init(props: Props) {
-            self.header = Header(with: props.style.header)
+            self.header = Header(
+                props: .init(
+                    title: "",
+                    effect: .none,
+                    endButton: .init(style: props.style.header.endButton, title: props.style.header.endButton.title),
```

Comment:
Same as above, I would suggest to avoid creating props inside, but form them along with WelcomeView.Props by view model. Does it make sense?

Reply:
done in 716775e7734ebaa6826ea65959b4b559ffeb70a1